### DISCUSSION TITLE
api description no longer available

### DISCUSSION THREAD

**Discussion by paradoxon82** [MiSchroe/klf-200-api#103] [Q&A] [answered]
Hi,
I tried to look up the API under [this link](https://www.velux.com/klf200)
But it seems they removed it. Is this available anywhere else?

**Comment by MiSchroe** [comment, marked as answer]
Hi,
you can find the API documentation at https://web.archive.org/web/20210210205126/https://velcdn.azureedge.net/-/media/com/api/klf200/technical%20specification%20for%20klf%20200%20api-ver3-18.pdf.
